addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -112,6 +126,11 @@ def _compute_display_name(self):
                 channel_name=member.channel_id.display_name,
             )
 
+    @api.depends("last_interest_dt", "unpin_dt", "channel_id.last_interest_dt")
+    def _compute_is_pinned(self):
+        for member in self:
+            member.is_pinned = not member.unpin_dt or (member.last_interest_dt and member.last_interest_dt > member.unpin_dt) or (member.channel_id.last_interest_dt and member.channel_id.last_interest_dt > member.unpin_dt)
```

Comment:
should apply formatter on new code, we should probably put these (as well as field definition) on multi line